### ISSUE TITLE
Remove the dynamo 0.1 API

### ISSUE THREAD

**Issue by arporter** [stfc/PSyclone#282]
Dynamo 0.1 has been superseded by 0.3 and has not been developed for several years.

**Comment by arporter** [comment]
Once #1100 is done, I shall remove all mention of the dynamo0.1 API from the docs.

**Comment by rupertford** [comment]
I thought I might join the party and remove dynamo0.1 API from the docs. Created branch [282_rm_dynamo0p1_docs](https://github.com/stfc/PSyclone/tree/282_rm_dynamo0p1_docs).
